plugins/repository-s3/generated-resources/plugin-descriptor.properties

```diff
@@ -36,7 +36,7 @@ classname=org.elasticsearch.plugin.repository.s3.S3RepositoryPlugin
 java.version=1.8
 #
 # 'elasticsearch.version' version of elasticsearch compiled against
-elasticsearch.version=3.0.0-SNAPSHOT
+elasticsearch.version=5.0.0-SNAPSHOT
```

Comment:
I think this file should be completely removed. Sounds like I added it by mistake with https://github.com/jasontedor/elasticsearch/commit/fb7723c1864c6489b5f8514668c31cec30cbb940#diff-bab62c6c0c1e7a3e2e833bc29d157280